libbeat/docs/shared-docker.asciidoc

```diff
@@ -100,6 +100,10 @@ conventional approach is to provide a configuration file via a volume mount, but
 it's also possible to create a custom image with your
 configuration included.
 
+// REVIEWERS: We don't have a journalbeat version of this file yet. Should I
```

Comment:
Is this file added manually? I see that this file is missing from https://github.com/elastic/beats/pull/9391.

Reply:
I think Dan added them manually. @DanRoscigno Do you want to add the config example, or should we do that?